integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/indexnull/IndexNullAsValueIT.java

```diff
@@ -74,6 +77,10 @@ void indexNullAsValue_spatial() {
 
 	@Test
 	void indexNullAsValue_vector() {
+		assumeTrue(
+				TckConfiguration.get().getBackendFeatures().supportsVectorSearchIndexNullsAs(),
+				"These tests only make sense for a backend where Vector Search is supported and implemented."
+		);
```

Comment:
Maybe test that we get a clear error message when it's *not* supported?

Or just remove the `indexNullAs` feature for vectors :)
Or handle it on the client side.

Reply:
added null-indexing into the codec.